code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -173,3 +173,11 @@
 	attribute_requirements = list(
 							JUSTICE_ATTRIBUTE = 60
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/heart
+	name = "bleeding heart"
+	desc = "And the prayer shall inevitably end with the eternal despair of its worshiper."
+	icon_state = "heart"
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 10, BLACK_DAMAGE = 10, PALE_DAMAGE = 50) // 140
```

Comment:
I don't think it should have such high red.
In LC, it was a tool that dealt *random* damage type, so we could probably interpret it here better?

Reply:
Not that I know aside from keeping 40/30 across the board and that would mess with the weapon. I chose high red to keep synergy with the weapon.

Reply:
Wouldn't it be funny if weapon dealt random damage too?